commercetools-models/src/test/java/io/sphere/sdk/carts/commands/CartDeleteCommandIntegrationTest.java

```diff
@@ -16,5 +24,18 @@ public void deleteCartWithDataErasureTests() {
         Cart cartQueried = client().executeBlocking(CartByIdGet.of(cart));
         Assertions.assertThat(cartQueried).isNull();
     }
-
-}
+    
+    @Test
+    public void deleteCartInStoreWithDataErasure() {
+        StoreFixtures.withStore(client(), store -> {
```

Comment:
Here it would be nice to use a static import - that would make it more consistent with the other code 😉